frontend/src/metabase/entities/tables.js

```diff
@@ -74,11 +74,11 @@ const Tables = createEntity({
 
   // ACTION CREATORS
   objectActions: {
-    updateProperty(entityObject, name, value, opts) {
+    updateProperty(entityObject, name, value, skipMerge, opts) {
```

Comment:
- 5 arguments here now
- in other places `skipMerge` is the last, optional argument, but here it's second to last, which makes it non-optional if `opts` is to be used